airflow/bin/cli.py

```diff
@@ -185,6 +185,56 @@ def trigger_dag(args):
     logging.info(message)
 
 
+def batch_set_pools(args):
+    """
+    set a batch of Airflow pools
+
+    :param args: arguments from argparser
+    :return:
+    """
+    session = settings.Session()
+    pool_names = args.names
+    pool_slots = args.slots
+    pool_descriptions = args.descriptions
+    print("Number of pool names input: {}".format(len(pool_names)))
+    print("Number of pool slots count input: {}".format(len(pool_slots)))
+    print("Number of pool description input: {}".format(len(pool_descriptions)))
+
+    if len(pool_names) != len(pool_slots) or len(pool_names) != len(pool_descriptions):
+        print("Numbers of names, pools, descriptions are not equal to each other")
+    else:
+        pool_metadata_by_name = dict()
```

Comment:
not a big deal, but metadata_by_pool seems to capture more what the variable is.